src/components/HeaderText/HeaderText.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from "prop-types";
+import Wavy from "./Wavy";
+
+const HeaderText = ({ text, wavyText, size, textColor, textAlign }) => {
+  let sizeClass;
+
+  switch (size) {
+    case "h1":
+      sizeClass = "max-[620px]:text-4xl text-6xl";
+      break;
+    case "h2":
+      sizeClass = "text-5xl";
+      break;
+    case "h3":
+      sizeClass = "text-4xl";
+      break;
+    default:
+      sizeClass = "text-3xl";
+  }
+
+  const headerClass = `${sizeClass} ${textColor === "white" ? "text-white" : "text-black"} ${textAlign} font-bold relative ${textAlign === "text-left" ? "inline-flex" : "flex"} flex-col`;
+
+  const splitText = text.split(wavyText);
+
+  return (
+    <div className={headerClass}>
+      <span>
+        <span className="leading-3">{splitText[0]}</span>
+        <span className="leading-3 relative inline-block">
+          {wavyText}
+          <Wavy textAlign={textAlign} size={size} />
+        </span>
+        <span className="leading-3">{splitText[1]}</span>
+      </span>
+    </div>
+  );
+};
+
+HeaderText.propTypes = {
+  text: PropTypes.string.isRequired,
+  wavyText: PropTypes.string.isRequired,
+  size: PropTypes.oneOf(["h1", "h2", "h3"]),
+  textColor: PropTypes.oneOf(["white", "black"]),
+  textAlign: PropTypes.oneOf(["text-left", "text-center", "text-right"]),
```

Comment:
This is not needed anymore.
In general please check the entire code as with the updated implementation you don't need all these alignmenyt logic.s

Also remove `classnames` dependency.

At least visit this link https://github.com/rt4914/KlinicCon-Frontend/pull/47/files to self-review this PR to make sure there are no unwanted changes.

Reply:
Yes, removed `alignment `dependency and `classnames `too.